plugins/kubernetes-common/src/permissions.ts

```diff
@@ -24,8 +24,28 @@ export const kubernetesProxyPermission = createPermission({
   attributes: {},
 });
 
+/** This permission is used to check access to the /resources and /services/:serviceId endpoints
+ * @public
+ */
+export const kubernetesResourcesPermission = createPermission({
+  name: 'kubernetes.resources',
+  attributes: {},
+});
+
+/** This permission is used to check access to the /clusters endpoint
+ * @public
+ */
+export const kubernetesClustersPermission = createPermission({
+  name: 'kubernetes.clusters',
+  attributes: {},
+});
```

Comment:
perhaps we can be more explicit:

```suggestion
export const kubernetesResourcesReadPermission = createPermission({
  name: 'kubernetes.resources.read',
  attributes: {
    action: 'read',
  },
});

/** This permission is used to check access to the /clusters endpoint
 * @public
 */
export const kubernetesClustersReadPermission = createPermission({
  name: 'kubernetes.clusters.read',
  attributes: {
    action: 'read',
  },
});
```